celer/group_fast.pyx

```diff
@@ -271,6 +281,7 @@ cpdef celer_grp(
         # elif pb == LOGREG:
             # radius = sqrt(gap / 2.) / alpha
 
+        # TODO potontially add weights
```

Comment:
Do we need to implement weights for ``set_prios_grp``?

Reply:
yes, as we do for set_prios